src/OSPSuite.Core/Domain/Services/ParameterIdentifications/ParameterIdentificationEngine.cs

```diff
@@ -102,6 +104,7 @@ private ParallelOptions createParallelOptions(CancellationToken token)
       public void Stop()
       {
          _cancellationTokenSource.Cancel();
```

Comment:
Interesting. That should be enough normally to cancel the task. Does this change make a difference

Reply:
cancellation token does not cancel the current run that's in progress

Reply:
oh yeah. Just the next one